feat: UIG-3185 - componenten geschrapt - verwijdering van de oude componenten waarvoor er een next- variant is

kspeltix opened the pull request:
https://www.milieuinfo.be/jira/browse/UIG-3185
https://www.milieuinfo.be/bamboo/browse/UIGOV-CUWC520

Goldflow committed:
feat: UIG-2868 - form-next - form-validation documentatie bijgewerkt

Goldflow committed:
feat: UIG-2868 - vl-form-demo - WCAG verbeteringen doorgevoerd

Het gedocumenteerde form patroon verbeterd door juiste placeholders in te vullen en labels aan te passen.

Goldflow committed:
feat: UIG-3075 - vl-form-control - aria-description van de form-control wordt nu aangepast op basis van de foutboodschap

Goldflow committed:
feat: UIG-3074 - vl-checkbox-next - WCAG verbeteringen doorgevoerd

Goldflow committed:
feat: UIG-3084 - vl-datepicker-next - WCAG verbeteringen doorgevoerd

Veranderingen voor focus, disabled, aria-labels op verschillende kind elementen, mobile variant doorgevoerd.
Storybook verbeterd & cypress testen uitgebreid.

Goldflow committed:
feat: UIG-3077, UIG-3076 - vl-input-field-next, vl-input-field-masked-next - WCAG verbeteringen doorgevoerd

Cypress testen uitgebreid

Goldflow committed:
feat: UIG-3078 - vl-radio-group-next - WCAG verbeteringen doorgevoerd

Fieldset patroon toegevoegd en aria-invalid attribuut toegepast op kind elementen wanneer dit relevant is.

Goldflow committed:
feat: UIG-3080 - vl-select-next, vl-select-rich-next - WCAG verbeteringen doorgevoerd

Goldflow committed:
feat: UIG-3081 - vl-textarea-next - WCAG verbeteringen doorgevoerd

Goldflow committed:
feat: UIG-3097 - vl-textarea-rich-next - WCAG verbeteringen doorgevoerd

Standaard stelt TinyMCE aria-hidden in op true na initialisatie, maar dit is problematisch voor screenreaders dus we verwijderen we dit attribuut na het initialiseren van TinyMCE.

Goldflow committed:
feat: UIG-3082 - vl-upload-next - WCAG verbeteringen doorgevoerd

De lijst van geüploade bestanden worden nu enkel opgelijst in een semantische lijst wanneer meer dan 1 bestand upgeload is. Dit omdat een lijst anders 4 onnodige stappen geeft aan iemand die een screenreader gebruikt.
Naar accessibility & UI raadt onze WCAG expert aan dat de lijst van bestanden boven de upload form control worden getoond in plaats van eronder, dit is dan bij deze ook zo aangepast.
Storybook verbeterd & cypress testen uitgebreid.

kspeltix committed:
feat: UIG-3185 - componenten geschrapt - verwijdering van de oude componenten waarvoor er een next- variant is

Goldflow committed:
feat: UIG-3199 - vl-button-next, vl-link-next - icon-placement verbeteringen doorgevoerd

Vroeger werd er standaard margin links of rechts van de icon ingesteld, nu wordt er enkel margin ingesteld als het icon-placement attribuut expliciet ingesteld wordt.
Enkele algemene verbeteringen doorgevoerd.
Storybook verbeterd & cypress testen uitgebreid.

Goldflow committed:
fix: .only vergetelheid verwijderd in test bestanden

Goldflow commented on libs/sections/src/cookie-consent/vl-cookie-consent-opt-in.section.ts:
```diff
@@ -43,7 +42,7 @@ export class VlCookieConsentOptIn extends BaseElementOfType(HTMLElement) {
 
     _labelChangedCallback(oldValue: string, newValue: string) {
-        this._checkboxElement.setAttribute('data-vl-label', newValue);
+        this._checkboxElement.innerHTML = newValue;
```
Gebruik hier geen `.innerHTML`, mogelijks security issues. Overweeg hier `.textContent` te gebruiken om gewoon tekst te vervangen.

Meer context: https://developer.mozilla.org/en-US/docs/Web/API/Node/textContent#differences_from_innerhtml

Ik bedenk me tegelijk ook dat het goed zou zijn dat we eventueel een attribuut hadden om beschrijving / inhoud van de checkbox in te vullen... of alternatief moeten we van de waarde van het `label`-attribuut in default slot steken van `vl-checkbox-next` (gelijkaardig zoals we recentelijk deden voor de `vl-error-message-next`)

kspeltix replied:
Kan best zijn, maar lijkt mij niets te maken te hebben met deze PR waar ik oude componenten schrap.

Goldflow reviewed (changes requested):
Ziet er goed uit. 2 opmerkingen:
- gebruik eerder `.textContent` ipv `.innerHTML` wanneer het kan (zie comment)
- verwijder `/toggle-button`-folder

Commit:
feat: UIG-3168 - vl-button-next - loading state hoogte verbeterd

Vroeger versprong de button in hoogte na toevoeging van het loading attribuut, nu blijft de hoogte dezelfde.
Cypress testen uitgebreid.

kspeltix committed:
feat: UIG-3142 - vl-select-richt-next - pijltjestoetsen veroorzaakten een tab-wissel bij integratie in de vl-tabs

k-o-e-n committed:
feat: UIG-3178 - vl-toaster - gebruikt nu z-index waarde uit DV

Aangepast voor het vl-toaster element via @domg/govflanders-style upgrade.
Aangepast voor het vl-toaster-next component via nieuwe z-index variabelen.
Andere z-index variabelen uit @domg/govflanders-style worden nu ook gebruikt waar mogelijk.

k-o-e-n committed:
feat: UIG-3206 - documentatie updates voor bijdragers

k-o-e-n committed:
feat: UIG-3147 - vl-group-next - `.vl-group-next--baseline` uitlijning toegevoegd

`.vl-group-next--baseline` uitlijning toegevoegd. Dit is een extra modifier die toelaat om de elementen in een vl-group-next component uit te lijnen op de tekst basislijn.

k-o-e-n committed:
feat: UIG-3179 - vl-modal - vl-action-group vervangen door vl-group-next

k-o-e-n committed:
feat: UIG-3148 - vl-popover - styling volgens DV, ondersteuning voor action-list dividers

kspeltix committed:
feat: UIG-3190 - vl-tabs - vl-click event bij tab klik

kspeltix committed:
feat: UIG-3204 - qlik componenten volledig verwijderd

k-o-e-n committed:
feat: UIG-3156 - vl-modal - sluiten adhv 'Escape' key verbeterd

`vl-modal` componenten met `closable` attribuut zijn sluitbaar met de "Escape"-key. Zonder `closable` of met `closable="false"` zijn deze niet sluitbaar met de "Escape"-key.
De focus wordt niet meer op de `dialog`-tag geplaatst bij het openen, maar blijft op het eerste interactieve element in de dialog.

kspeltix committed:
feat: UIG-3211 - vl-tab-next - introductie naïeve next component (wegwerken elementen)

semantic-release-bot committed:
chore(release): 1.44.0 [skip ci]

# [1.44.0](https://github.com/milieuinfo/uigov-web-components/compare/v1.43.1...v1.44.0) (2025-02-19)

### Bug Fixes

* .only vergetelheid verwijderd in test bestanden ([bdf422b](https://github.com/milieuinfo/uigov-web-components/commit/bdf422baca30786bbb7d008627bc24a847989a1b))

### Features

* UIG-3142 - vl-select-richt-next - pijltjestoetsen veroorzaakten een tab-wissel bij integratie in de vl-tabs ([43787e6](https://github.com/milieuinfo/uigov-web-components/commit/43787e65a0160490e7b353c88ddd04a4babe8007))
* UIG-3144 - vl-map - map disposen in disconnectedCallback ([85bbd07](https://github.com/milieuinfo/uigov-web-components/commit/85bbd077353c4f0c993d2db11590d85e3c434981))
* UIG-3147 - vl-group-next - `.vl-group-next--baseline` uitlijning toegevoegd ([43fe6c1](https://github.com/milieuinfo/uigov-web-components/commit/43fe6c116bc8053afc1f00aec662bb686c8d89d3))
* UIG-3148 - vl-popover - styling volgens DV, ondersteuning voor action-list dividers ([c2f3d6c](https://github.com/milieuinfo/uigov-web-components/commit/c2f3d6c433778c946bab287f4a05683bc9e004f6))
* UIG-3156 - vl-modal - sluiten adhv 'Escape' key verbeterd ([d9d5790](https://github.com/milieuinfo/uigov-web-components/commit/d9d5790f9780d131f05fe16906fcca0c65ba8da3))
* UIG-3168 - vl-button-next - loading state hoogte verbeterd ([323b2a3](https://github.com/milieuinfo/uigov-web-components/commit/323b2a32acfe8af11ab1df68153ac504df18b66b))
* UIG-3178 - vl-toaster - gebruikt nu z-index waarde uit DV ([aa11513](https://github.com/milieuinfo/uigov-web-components/commit/aa11513097c14fb307af300236ae6758fb3a43d4))
* UIG-3179 - vl-modal - vl-action-group vervangen door vl-group-next ([84ac67b](https://github.com/milieuinfo/uigov-web-components/commit/84ac67b46cba82f123dabd71b35a858b17863955))
* UIG-3190 - vl-tabs - vl-click event bij tab klik ([aeafe92](https://github.com/milieuinfo/uigov-web-components/commit/aeafe92be32388ec9b0515a51bdb6194f10f0894))
* UIG-3192 - vl-button-next - gebruik van vl-toggle-button vervangen door vl-button-next ([99657f6](https://github.com/milieuinfo/uigov-web-components/commit/99657f6172da09201e4395dcce2f9a9dbc105363))
* UIG-3196 - vl-accessibility - vl-visually-hidden css introductie ([6708990](https://github.com/milieuinfo/uigov-web-components/commit/670899000ba68d5447d6fafeebf5ad73f452f98f))
* UIG-3198 - vl-color-raw - consistente kleur variabelen ([2d31d67](https://github.com/milieuinfo/uigov-web-components/commit/2d31d67acb88c4a0efa98c3ec05f39664e936c77))
* UIG-3199 - vl-button-next, vl-link-next - icon-placement verbeteringen doorgevoerd ([df607b6](https://github.com/milieuinfo/uigov-web-components/commit/df607b6d38d318e97b7d46fd0347794ccbc873d4))
* UIG-3201 - vl-error-message-next - mogelijkheid meerdere lijnen toegevoegd ([ece7bc1](https://github.com/milieuinfo/uigov-web-components/commit/ece7bc1072691ea8678880b74b314e0404dfdd94))
* UIG-3204 - qlik componenten volledig verwijderd ([2329ff6](https://github.com/milieuinfo/uigov-web-components/commit/2329ff63b3bb5e9a9eb5844cb917392d49720245))

k-o-e-n committed:
feat: UIG-3212 - vl-button-next - ghost variant aangemaakt

k-o-e-n committed:
feat: UIG-3150 - vl-info-tile - extra slot toegevoegd voor menu

k-o-e-n committed:
feat: UIG-3151 - vl-select-next - ellipsis toegevoegd bij lange opties

Goldflow committed:
feat: UIG-3164 - vl-upload-next - upload progress event & callback voorzien

k-o-e-n committed:
feat: UIG-3184 - vl-datepicker - automatische positionering hersteld en extra attributen "position" en "static"

k-o-e-n committed:
feat: UIG-3141 - vl-map - het selectiekader is terug zichtbaar bij de delete action

k-o-e-n committed:
feat: UIG-3208 - vl-search-filter - Escape functionaliteit verplaatst naar vl-rich-data

kspeltix committed:
fix: UIG-3218 - Storybook - 'Show Code' was gebroken door onnodige package upgrades

Goldflow committed:
fix: UIG-3222 - vl-error-message-next - pre-line wordt nu enkel met attribuut gestuurd

k-o-e-n committed:
feat: UIG-3219 - vl-pill - is="vl-button-pill" is nu <vl-pill clickable>

k-o-e-n committed:
feat: UIG-3220 - vl-select-location-next - introductie component

Goldflow committed:
fix: UIG-2929 - vl-search-filter-next - footer styling rechtgezet

semantic-release-bot committed:
chore(release): 1.45.0 [skip ci]

# [1.45.0](https://github.com/milieuinfo/uigov-web-components/compare/v1.44.0...v1.45.0) (2025-03-14)

### Bug Fixes

* UIG-2929 - vl-search-filter-next - footer styling rechtgezet ([ed99ee7](https://github.com/milieuinfo/uigov-web-components/commit/ed99ee7de5c22c12b19a3372dd0069dc12f6fe54))
* UIG-3218 - Storybook - 'Show Code' was gebroken door onnodige package upgrades ([746b5ac](https://github.com/milieuinfo/uigov-web-components/commit/746b5ac0e36b5884e94612ee6c77c454e306f5cc))
* UIG-3222 - vl-error-message-next - pre-line wordt nu enkel met attribuut gestuurd ([00c2847](https://github.com/milieuinfo/uigov-web-components/commit/00c2847d30a3493550ca893d6b5c2e5a965eeee2))

### Features

* UIG-2868 - form-next - form-validation documentatie bijgewerkt ([3521905](https://github.com/milieuinfo/uigov-web-components/commit/352190567eb1778345b8c9a13f3aa647d5da144c))
* UIG-2868 - vl-form-demo - WCAG verbeteringen doorgevoerd ([b256c95](https://github.com/milieuinfo/uigov-web-components/commit/b256c953528a06f42422aa9a546482fa2dbd9fb7))
* UIG-3074 - vl-checkbox-next - WCAG verbeteringen doorgevoerd ([4cbeb4c](https://github.com/milieuinfo/uigov-web-components/commit/4cbeb4c33a9d786c2ba0dbd5cb87fb8a32c55107))
* UIG-3075 - vl-form-control - aria-description van de form-control wordt nu aangepast op basis van de foutboodschap ([d0757d6](https://github.com/milieuinfo/uigov-web-components/commit/d0757d6487a5059481421123becee170da20393d))
* UIG-3077, UIG-3076 - vl-input-field-next, vl-input-field-masked-next - WCAG verbeteringen doorgevoerd ([2591f81](https://github.com/milieuinfo/uigov-web-components/commit/2591f811676d19fcffe9a82bcd4d3408a4d0123b))
* UIG-3078 - vl-radio-group-next - WCAG verbeteringen doorgevoerd ([83fdfe3](https://github.com/milieuinfo/uigov-web-components/commit/83fdfe3d86db2d3b6efad79b03d1cfb8092f0056))
* UIG-3080 - vl-select-next, vl-select-rich-next - WCAG verbeteringen doorgevoerd ([7b0b779](https://github.com/milieuinfo/uigov-web-components/commit/7b0b779b3b9ef62e6880fbe7d3e6feec66cb4650))
* UIG-3081 - vl-textarea-next - WCAG verbeteringen doorgevoerd ([c22d331](https://github.com/milieuinfo/uigov-web-components/commit/c22d3311c0166f86eb347b65030a4d68bc7acf2f))
* UIG-3082 - vl-upload-next - WCAG verbeteringen doorgevoerd ([0db92bb](https://github.com/milieuinfo/uigov-web-components/commit/0db92bb8637d3ae84eccb9a5257cf5e59aadf873))
* UIG-3084 - vl-datepicker-next - WCAG verbeteringen doorgevoerd ([f2a6926](https://github.com/milieuinfo/uigov-web-components/commit/f2a692684089a7da67bde7db941a76cfd9189240))
* UIG-3097 - vl-textarea-rich-next - WCAG verbeteringen doorgevoerd ([9fac920](https://github.com/milieuinfo/uigov-web-components/commit/9fac920c8dc1acab62ef79a453759a1d4fb9840a))
* UIG-3141 - vl-map - het selectiekader is terug zichtbaar bij de delete action ([6aa526d](https://github.com/milieuinfo/uigov-web-components/commit/6aa526d3879d3aa6bed7607b797f30127d4b91a7))
* UIG-3150 - vl-info-tile - extra slot toegevoegd voor menu ([a823d08](https://github.com/milieuinfo/uigov-web-components/commit/a823d0896a4735743e409206d32de0f0e200ea48))
* UIG-3151 - vl-select-next - ellipsis toegevoegd bij lange opties ([12a9ce8](https://github.com/milieuinfo/uigov-web-components/commit/12a9ce827b29d70b123ffd67b1b3b1b996cd986c))
* UIG-3164 - vl-upload-next - upload progress event & callback voorzien ([3c6f13b](https://github.com/milieuinfo/uigov-web-components/commit/3c6f13b931f86e36f7d4d566ec1fb4d525174548))
* UIG-3184 - vl-datepicker - automatische positionering hersteld en extra attributen "position" en "static" ([f4b6c03](https://github.com/milieuinfo/uigov-web-components/commit/f4b6c03433edff4b7aa7b5adeaaeb9c4eea2003f))
* UIG-3208 - vl-search-filter - Escape functionaliteit verplaatst naar vl-rich-data ([f3c2914](https://github.com/milieuinfo/uigov-web-components/commit/f3c29147f9faa72189fdb34ac771a55307b63ca9))
* UIG-3211 - vl-tab-next - introductie naïeve next component (wegwerken elementen) ([14b3573](https://github.com/milieuinfo/uigov-web-components/commit/14b35737336faf8a7fb2b93b000f45f30a5ff7ff))
* UIG-3212 - vl-button-next - ghost variant aangemaakt ([dc05789](https://github.com/milieuinfo/uigov-web-components/commit/dc057899feb80d03f9dcdc7ad3d11da504a7945d))
* UIG-3219 - vl-pill - is="vl-button-pill" is nu <vl-pill clickable> ([73348d8](https://github.com/milieuinfo/uigov-web-components/commit/73348d8e0d845f23f2c09ce3ac1b2c410e286695))
